### ISSUE TITLE
Create a sphinx document 

### ISSUE THREAD

**Issue by willingc** [calpolydatascience/jupyterhub-deploy-data301#6]
@ellisonbg I would like to create a Sphinx document for this repo. My hope is that it could be used as one of the use cases for a JupyterHub deployment. It also might be a good first project for the summer interns. I will set up the structure and let them work on migrating the content from the readme into a doc that we could deploy on RTD. Thoughts?

**Comment by ellisonbg** [comment]
In talking with @minrk at the dev meeting our plan is move this repo over
to the jupyter or jupyterhub org and star to use it as one of the official
supported deployment scenarios. Given that, I think your proposal makes
tons of sense. @minrk - what is the status on the jupyterhub org? Can we
move this over to one of the jupyterorgs now? @willingc - in the meantime,
feel free to submit any PRs related to documentation.

On Wed, Apr 13, 2016 at 1:17 PM, Carol Willing <notifications@github.com>
wrote:

> @ellisonbg <https://github.com/ellisonbg> I would like to create a Sphinx
> document for this repo. My hope is that it could be used as one of the use
> cases for a JupyterHub deployment. It also might be a good first project
> for the summer interns. I will set up the structure and let them work on
> migrating the content from the readme into a doc that we could deploy on
> RTD. Thoughts?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/calpolydatascience/jupyterhub-deploy-data301/issues/6>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com

**Comment by minrk** [comment]
@ellisonbg I made the org, but haven't created any repos. The prospect of moving jupyter/jupyterhub is making me a little squeamish.

**Comment by minrk** [comment]
@ellisonbg I made the repo here: https://github.com/jupyterhub/jupyterhub-deploy-nbgrader

Shall I just push the current state of this repo, and start from there?

**Comment by ellisonbg** [comment]
Yes, unless you are having second thoughts about moving everything to the new org...your call

**Comment by minrk** [comment]
I think it will be good in the long run, I've just encountered more things than I'd like that wget from JupyterHub URLs in the last few days.

**Comment by ellisonbg** [comment]
OK, sounds good. Go ahead and move the ansible stuff over then...

On Thu, Apr 14, 2016 at 7:20 AM, Min RK <notifications@github.com> wrote:

> I think it will be good in the long run, I've just encountered more things
> than I'd like that wget from JupyterHub URLs in the last few days.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/calpolydatascience/jupyterhub-deploy-data301/issues/6#issuecomment-209966805>
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com
